smart-contract-verifier/src/metrics.rs

```diff
@@ -23,4 +25,38 @@ lazy_static! {
         "contract compilation time in seconds",
     )
     .unwrap();
+    pub static ref COMPILES_IN_FLIGHT: Gauge = register_gauge!(
```

Comment:
Maybe `COMPILATIONS_IN_FLIGHT` (the same for below metrics)? It seems that we are using nouns for metrics, while _compile_ is definitely a verb. 

It seems that `COMPILE_TIME` above is also wrong and should be `COMPILATION_TIME` but I believe it is already too late to fix in that PR as it is already in use. But one incorrect value _should not_ be a reason to continue using invalid words, imo

Reply:
agree to change to COMPILATIONS_IN_FLIGHT, but COMPILE_TIME is ok for me, I don't know why you think it's wrong